backend/src/ee/services/secret-snapshot/secret-snapshot-service.ts

```diff
@@ -370,7 +371,20 @@ export const secretSnapshotServiceFactory = ({
         const secrets = await secretV2BridgeDAL.insertMany(
           rollbackSnaps.flatMap(({ secretVersions, folderId }) =>
             secretVersions.map(
-              ({ latestSecretVersion, version, updatedAt, createdAt, secretId, envId, id, tags, ...el }) => ({
+              ({
+                latestSecretVersion,
+                version,
+                updatedAt,
+                createdAt,
+                secretId,
+                envId,
+                id,
+                tags,
+                userActorId,
+                identityActorId,
+                actorType,
```

Comment:
might be worth adding a comment here to note that we are intentionally omitting these fields during the secret v2 insertion